rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-galera.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Confirm wsrep_cluster_size is equal to the amount of galera containers
+  shell: "mysql -e \"show status like 'wsrep_cluster_size'\" | grep wsrep_cluster_size | awk '{print $2}'"
+  register: mysql_cluster_size
+  failed_when: "'{{ groups['galera_all']|length }}' not in mysql_cluster_size.stdout"
+  changed_when: mysql_cluster_size.rc != 0
```

Comment:
couldn't this be a part of the failed_when: ?
failed_when: "groups['galera_all']|length not in mysql_cluster_size.stdout or mysql_cluster_size.rc != 0" ?